cypress/integration/app/activateuser.spec.ts

```diff
@@ -0,0 +1,104 @@
+describe("Invite user", () => {
+  it("Enable SMTP setting", () => {
+    cy.setup();
+    cy.login();
+
+    cy.visit("/settings/organization");
+
+    cy.findByLabelText(/enable smtp/i).check({ force: true });
+
+    cy.findByLabelText(/sender address/i)
+      .click()
+      .type("rachel@fleetdm.com");
+
+    cy.findByLabelText(/smtp server/i)
+      .click()
+      .type("localhost");
+
+    cy.get("#port").click().type("{selectall}{backspace}1025");
+
+    cy.findByLabelText(/use ssl\/tls/i).check({ force: true });
+
+    cy.findByLabelText(/smtp username/i)
+      .click()
+      .type("rachelsusername");
+
+    cy.findByLabelText(/smtp password/i)
+      .click()
+      .type("rachelspassword");
+
+    cy.findByRole("button", { name: /update settings/i }).click();
+  });
+
+  it("Invite user", () => {
```

Comment:
Let's merge this all into a single test (`it` block), following the best practice described in https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests.

Reply:
Thanks for the reference, all great to know.